app/controllers/concerns/sync_resource_translation_strings.rb

```diff
@@ -9,17 +9,24 @@ module SyncResourceTranslationStrings
 
   def sync_translatable_resource_strings
     translatable_resources.each do |resource|
-      TranslationSyncWorker.perform_async(
-        resource.class.name,
-        resource.id,
-        resource.translatable_language
-      )
+      # NOTE: In Rails 5 this method is deprecated, and we should use
+      # resource.saved_changes instead.
+      if (resource.previous_changes.keys & resource.class.translatable_attributes.map(&:to_s)).present?
```

Comment:
super minor comment - perhaps extract this to a method with name `translations_have_changed?` for readability?